src/content/tutorial/1-basics/1-introduction/3-one-way-data-flow/content.md

```diff
@@ -0,0 +1,91 @@
+---
+type: lesson
+title: One-way Data Flow
+focus: /index.html
+---
+
+# One-way Data Flow
+
+Signals can be used to create UIs with one-way data flow,
+where data flows top-down through the view hierarchy
+and the UI is computed as a pure function of this state.
+Popularized by React, this is commonly expressed as `ui = f(state)`.
+
+_In this lesson we'll refactor our counter to be created as a pure function of state_
```

Comment:
I think we could phrase this in isolation, or avoid refactoring -- to me, it feels like an extra step